config.rb

```diff
@@ -0,0 +1,4 @@
+module URLS
+  BASE_URL='https://openai.com'
```

Comment:
Add constant in Scraper class, it is better than in Module, for now.

Reply:
i don't really understand what do you mean, earlier you told that we need to move URLs in another file. And now i just need to set a constant variable in Scraper class. Am i wrong?